### DISCUSSION TITLE
How do i download the .gp file ?

### DISCUSSION THREAD

**Discussion by Draconic112** [noahmaranesi/Ultimate-Guitar-Hack#5] [Q&A] [answered]
Hi, I have just downloaded the zip file, and there is only a readme file with a .gitignore . There is no explanation on how to download the gp file. Could you help me pls?
Thxxx !

**Comment by noahmaranesi** [comment, marked as answer]
Hi @Draconic112, there is nothing to download anymore. Clicking on Code → Download ZIP will download the entire project as is — which is only two files.

[If something is made available again in the future](https://github.com/noahmaranesi/Ultimate-Guitar-Hack#how-to), it can be found within [Releases](../releases)

**Reply by Draconic112** [reply]
Oh okay
And is there still a way to download official tabs?
Sorry if I'm being repetitive 😅

**Reply by jinxdefusing** [reply]
There was a user script up on greasy fork. It worked flawlessly with the tampermonkey plugin on firefox for well over a year. Stopped working sometime last year. AFAIK, the dead script is gone forever. I don't think we'll see anything posted here either. It's possible that a new workaround will crop up somehwere, but it will probably come from another source.
